platform/android/plugin/godot_plugin_config.h

```diff
@@ -67,7 +50,24 @@ The `dependencies` section and fields are optional and defined as follow:
 
  See https://github.com/godotengine/godot/issues/38157#issuecomment-618773871
  */
-struct PluginConfig {
+struct PluginConfigAndroid {
+	static const char *PLUGIN_CONFIG_EXT;
```

Comment:
You should be able to have the values declared inline with C++11 no?

Reply:
No, got inline initializer error on compilation, so left it as it is now.

Reply:
According to https://stackoverflow.com/a/1563906 this may work with `inline` in C++17 (which we use).

Reply:
Confirmed that it builds fine (with GCC at least):
```diff
diff --git a/platform/android/plugin/godot_plugin_config.h b/platform/android/plugin/godot_plugin_config.h
index 04091b49f7..a2ea42d311 100644
--- a/platform/android/plugin/godot_plugin_config.h
+++ b/platform/android/plugin/godot_plugin_config.h
@@ -51,7 +51,7 @@ The `dependencies` section and fields are optional and defined as follow:
  See https://github.com/godotengine/godot/issues/38157#issuecomment-618773871
  */
 struct PluginConfigAndroid {
-       static const char *PLUGIN_CONFIG_EXT;
+       inline static const char *PLUGIN_CONFIG_EXT = ".gdap";
 
        static const char *CONFIG_SECTION;
        static const char *CONFIG_NAME_KEY;
@@ -84,8 +84,6 @@ struct PluginConfigAndroid {
        Vector<String> custom_maven_repos;
 };
 
-const char *PluginConfigAndroid::PLUGIN_CONFIG_EXT = ".gdap";
-
 const char *PluginConfigAndroid::CONFIG_SECTION = "config";
 const char *PluginConfigAndroid::CONFIG_NAME_KEY = "name";
 const char *PluginConfigAndroid::CONFIG_BINARY_TYPE_KEY = "binary_type";
```